doc/ug/getting_started_fpga.md

```diff
@@ -47,56 +48,85 @@ The `--build` argument is optional and ensures that the boot ROM is rebuilt for
 Alternatively, the boot ROM can be manually regenerated using the previous command.
 
 
-In the following example we synthesize the Earl Grey design for the Nexys Video board using Xilinx Vivado 2020.1.
+In the following example we synthesize the Earl Grey design for the ChipWhisperer CW310 board using Xilinx Vivado 2020.1.
+To target the Nexys Video board, replace `cw310` by `nexysvideo` in the instructions below.
 
 ```console
 $ . /tools/xilinx/Vivado/2020.1/settings64.sh
 $ cd $REPO_TOP
 $ ./meson_init.sh
 $ ./hw/top_earlgrey/util/top_earlgrey_reduce.py
 $ ninja -C build-out all
-$ fusesoc --cores-root . run --flag=fileset_top --target=synth lowrisc:systems:chip_earlgrey_nexysvideo
+$ fusesoc --cores-root . run --flag=fileset_top --target=synth lowrisc:systems:chip_earlgrey_cw310
 ```
 The `fileset_top` flag used above is specific to the OpenTitan project to select the correct fileset.
 
-The resulting bitstream is located at `build/lowrisc_systems_chip_earlgrey_nexysvideo_0.1/synth-vivado/lowrisc_systems_chip_earlgrey_nexysvideo_0.1.bit`.
+The resulting bitstream is located at `build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/lowrisc_systems_chip_earlgrey_cw310_0.1.bit`.
 See the [reference manual]({{< relref "ref_manual_fpga.md" >}}) for more information.
 
-## Connecting the board
+## Connecting the ChipWhisperer CW310 board
+
+The ChipWhisperer CW310 board supports different power options.
+It is recommended to power the board via the included DC power adapter.
+To this end:
+1. Set the *SW2* switch (to the right of the barrel connector) up to the *5V Regulator* option.
+1. Set the switch below the barrel connector to the right towards the *Barrel* option.
+1. Set the *Control Power* switch (bottom left corner, *SW7*) to the right.
+1. Ensure the *Tgt Power* switch (above the fan, *S1*) is set to the right towards the *Auto* option.
+1. Plug the DC power adapter into the barrel jack (*J11*) in the top left corner of the board.
+1. Use a USB-C cable to connect your PC with the *USB-C Data* connector (*J8*) in the lower left corner on the board.
+
+You can now use `dmesg` to determine which serial ports have been assigned.
+They should be named `'/dev/ttyACM*'`, e.g. `/dev/ttyACM1`.
+To ensure that you have sufficient access permissions to the devices, check `ls -l /dev/ttyACM*`.
```

Comment:
This guidance is misleading. Let's remove them and only point to the udev rules.

Even if you have access to the /dev/tty* device file, the chipwhisperer tooling will still fail with a beautiful chain of exceptions around "langid". I didn't fully debug it ultimately, but the user needs permissions not only to the TTY devices (listed here), but to the underlying USB devices (i.e. one layer down in the driver hierarchy), because they are used for probing for connected devices.

Reply:
That's changed now as well. I don't remember exactly but I thought I had these problems as well but they went away after re-plugging the boards once udev rules are set up.